internal/cri/server/images/image_pull.go

```diff
@@ -235,13 +235,12 @@ func (c *CRIImageService) PullImage(ctx context.Context, name string, credential
 		}
 	}
 
-	const mbToByte = 1024 * 1024
-	size, _ := image.Size(ctx)
-	imagePullingSpeed := float64(size) / mbToByte / time.Since(startTime).Seconds()
+	sizeByte, _ := image.Size(ctx)
+	imagePullingSpeed := float64(sizeByte) / time.Since(startTime).Seconds()
 	imagePullThroughput.Observe(imagePullingSpeed)
```

Comment:
Assumed to be MB 

https://github.com/containerd/containerd/blob/45e30913bc75c43148e00aade10891e5a4a6eba5/internal/cri/server/images/metrics.go#L27-L28

> 	// image size in MB / image pull duration in seconds

Reply:
ah, good catch, closing !